packages/core/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -169,7 +171,7 @@ const InfoCard: FC<Props> = ({
   }
 
   return (
-    <Card style={calculatedStyle}>
+    <Card style={calculatedStyle} className={className}>
```

Comment:
Just noting that this opens up for a lot of custom styling. @katz95 what's your view, do we want people to be able to customize the look and feel of the InfoCard?